### DISCUSSION TITLE
I have an Github Copilot Business Plan that doesn't allow Chat on JetBrains IDE

### DISCUSSION THREAD

**Discussion by anandnatampalli** [community/community#88081] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

We have an organisation account for github copilot, and under teams, policies we have enabled copilot chat for team members. 
Yet when I open the chat it says I can join the waitlist for individual subscribers.
**The chat works fine on VSCode though.**
See attached Screenshots:
* The organisation setting for github copilot business:
<img width="755" alt="Org Chat Enabled" src="https://github.com/community/community/assets/156316675/c488573b-c381-484a-b29f-53ba06992da2">  
*  Here is the organisation setting on my personal settings page showing that the chat is enabled.
<img width="738" alt="github account page" src="https://github.com/community/community/assets/156316675/8cb4300e-7c01-47b9-b8cd-fb0ece004504">  
*  Finally the Jetbrains IDE chat showing that I join the waitlist:
<img width="598" alt="JetBrains Ide shows waitlist" src="https://github.com/community/community/assets/156316675/8f0198f7-7f20-44d6-baaf-e083379f131b">

**Comment by DOS-SOCOMEC** [comment, marked as answer]
For Jetbrains Chat is not in public beta but private beta, so sadly we ask to join a waitlist ...

**Reply by anandnatampalli** [reply]
But the waitlist is only for individual subscription, I have a Business Subscription. There is no way to join the waitlist with a business subscription.

**Reply by gtaborcrossword** [reply]
I experienced the same today. Chat is enabled in settings, users see waitlist link but it directs to a page saying that it should be enabled in settings. Loop! Is it resolved for you now?

**Reply by nixon1333** [reply]
I am experiencing same too .

**Reply by bambamboole** [reply]
> If you are an owner of an invoiced organization or enterprise with a Copilot Business subscription and would like to participate in the beta for GitHub Copilot Chat in a JetBrains IDE, you can contact your account manager or solutions engineer to request access. Customers on Free, Team or self-serve Enterprise plans who do not have an account manager are not eligible for the private beta at this time.

Thats written here: https://docs.github.com/en/copilot/github-copilot-chat/using-github-copilot-chat-in-your-ide?tool=jetbrains_beta

SO if you do not have an account manager, you have to wait. Thats really as shame...

**Reply by martin-gunther-loop** [reply]
But why can I use it with PHPStorm with the same account, but not web storm, rider, and datagrip. It's ridiculous tbh

**Comment by goldenSebac** [comment]
We have exactly the same issue with our business subscription... 
I waited for 2 days but nothing.
Bug ?
![image](https://github.com/community/community/assets/18516197/17f26cf4-7a35-49e8-8237-423e6d2834b7)
![image](https://github.com/community/community/assets/18516197/0dfdf0a3-a402-4852-b17e-1b8c5e1238e0)

**Reply by davecheney** [reply]
Jetbrains chat remains a waitlisted feature because of the high level of uptake. Invitations are being sent as capacity permits.

**Comment by martin-gunther-loop** [comment]
Well I'm nearly at the end of my months trial having used the Jetbrains AI trial before. I guess I will now cancel github copilot and purchase the Jetbrains built in one itself, as have waited weeks now after joining the wait list.

Strange as I can access the chat in PHPStorm, but not Rider, WebStorm or Datagrip.

**Comment by Yerock09** [comment]
Creo que tú xke no te hice nada para que estés haciendo esto igual ni modo
